utils/src/ooinstall/oo_config.py

```diff
@@ -36,19 +36,24 @@ def __init__(self, **kwargs):
         self.public_ip = kwargs.get('public_ip', None)
         self.public_hostname = kwargs.get('public_hostname', None)
         self.connect_to = kwargs.get('connect_to', None)
+        self.run_on = kwargs.get('run_on', None)
 
         # Should this host run as an OpenShift master:
         self.master = kwargs.get('master', False)
 
         # Should this host run as an OpenShift node:
         self.node = kwargs.get('node', False)
+
+        # Should this host run as an HAProxy:
+        self.ha_proxy = kwargs.get('ha_proxy', False)
```

Comment:
I would lean towards master_lb as well..  tying the lb variable to a particular technology is just asking for us to eventually be deploying nginx using ha_proxy.